torchquad/integration/utils.py

```diff
@@ -206,3 +207,19 @@ def _torch_repeat(a, repeats, axis=None):
     # torch.repeat_interleave corresponds to np.repeat and should not be
     # confused with torch.Tensor.repeat.
     return torch.repeat_interleave(a, repeats, dim=axis)
+
+@partial(register_function, "torch", 'expand_dims')
+def _torch_expand_dims(a, axis):
+    import torch
+
+    return torch.unsqueeze(a, axis)
+
+def expand_func_values_and_squeeze_intergal(f):
+    def wrap(*args, **kwargs):
+        # i.e we only have one dimension, or the second dimension (that of the integrand) is 1
+        is_1d = len(args[1].shape) == 1 or (len(args[1].shape) == 2 and args[1].shape[1] == 1)
+        if is_1d:
+            warnings.warn("DEPRECATION WARNING: In future versions of torchquad, an array-like object will always be returned.")
```

Comment:
I think in all other places we use logger.warning if I am not mistaken? If so please use that here as well :pray: